### DISCUSSION TITLE
How do you handle synonyms in your dictionary?

### DISCUSSION THREAD

**Discussion by rolfmeles** [DraqueT/PolyGlot#1333] [General]
I am curouos on how you use the lexicon in case of synonyms in your language.

When you have one word in your native language and your conlang has several translations for it, or if you have several words in your native language that all translate to one same word in your conlang, how do you handle this in PolyGlot?

Do you
a) create separate lexicon entries for each translation, or
b) create one lexicon entry and put in all possible translations into the corresponding field, e. g. comma separated?

When yousing the search/filter function, both methods get you the output. So at first glance, they seem to be equally good. But is there any reason why you do it one and not the other way around?

I guess in a database you should create separate entries and only put in one meaning per entry. But should you handle PolyGlot as a database in this regard?

**Comment by TrapinchO** [comment]
I used multiple entries. First reason was that the translations had different class (noun and adjective) and the second was ease of native-conlang dictionary. If you had them in one field, you would have to search for the first term (ignoring the option to search in text), but this way you  can find just the one you need.

If the words are similar, I usually put the closest one into the native equivalent and then the rest into the description.
